### DISCUSSION TITLE
The link to Solutions for the Day 2 Morning point to the afternoon solutions

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1234] [General]
It's trivial but the link at the end of 19.1 (the Library exercise from Day 1 morning) is:

http://localhost:3000/exercises/day-2/solutions-afternoon.html#designing-a-library

whereas it should be

http://localhost:3000/exercises/day-2/solutions-morning.html#designing-a-library

**Comment by mgeisler** [comment]
Thanks @njr0 for spotting this! I think the link should just go away since the other exercises don't have such links: https://github.com/google/comprehensive-rust/pull/1235.

**Comment by njr0** [comment]
Sure! Thanks.

**Comment by djmitche** [comment]
In v2, I'm organizing the solutions as sub-pages of the exercises:
![image](https://github.com/google/comprehensive-rust/assets/28673/fbaaff89-2e27-4980-898c-06e46f571264)

**Reply by mgeisler** [reply]
Yeah, I like that — especially since the course is voluntary and without a grade etc... so people can "cheat" as much as they like.

**Reply by njr0** [reply]
Looks good to me.
